src/components/PrizePage/Prizes.js

```diff
@@ -64,11 +71,44 @@ const TableCell = withStyles({
 })(MuiTableCell);
 
 const useStyles = makeStyles({
+	prizeGrid: {
+		textAlign: 'center'
+	},
+	prizeCard: {
+		backgroundColor: '#e6e6e6',
```

Comment:
This color seems to be previously unused. Consider using grey[200] or grey[400], where grey is defined as:
https://github.com/uclaacm/hoth.uclaacm.com/blob/fc6a4e1036cf531e5c325ee4ff0c205702d8514c/src/components/layout.js#L13

e6e6e6 is closest to grey[300], so if grey[200] or grey[400] don't work very well let's use grey[300], which is still better than hardcoding the color.

Same for imageBox.borderColor.